### ISSUE TITLE
TB-Gateway Configuration issue

### ISSUE THREAD

**Issue by ChSM56** [thingsboard/thingsboard#421]
Hi

I have Installed the Server platform successfully with HSQLDb and been able to work around the platform. I am now facing issue of setting up Gateway working.
My Gateway on my portal after deleting all Demo devices looks like -

![image](https://user-images.githubusercontent.com/29800826/32884672-20e7d6d6-cae1-11e7-8a2b-3fe6125b3457.png)

I have taken the Access Token value from this screen 
![image](https://user-images.githubusercontent.com/29800826/32884724-448c288a-cae1-11e7-853b-36b3e4085a36.png)

I have populated the Token value per screen shot in the config file -
![image](https://user-images.githubusercontent.com/29800826/32884756-6b33b322-cae1-11e7-9982-11ae32ae968c.png)

My efforts to get going with Gateway are facing this challenge -
![image](https://user-images.githubusercontent.com/29800826/32884816-993e1f0a-cae1-11e7-8ace-af196693a04e.png)

Please help.

Thanks & Regards

**Comment by ssmaurya** [comment]
Add a space between colon and your access token and then try.

**Comment by ChSM56** [comment]
Thank you ssmaurya.
Besides, I noticed that I had not installed Mosquitto MQTT  so I had done that we well. 
Further, I changed the port for MQTT to 1882 from 1883.

Both the Gateway as well as the portal are running on the same System.

Here is to show that my Gateway connected to the portal -

![image](https://user-images.githubusercontent.com/29800826/32990130-ec29d1d6-cd49-11e7-903f-bca250121ba3.png)

Thank you so much.
